### DISCUSSION TITLE
a suggestion

### DISCUSSION THREAD

**Discussion by AlbertFnWesker** [IntelOrca/biorand#737] [General]
### Version

3.1.8 (a8420af)

### Seed

R721-EE9T-XNHH4077Z9Y0003X9KW9M8K6GENCNVWNYCVV6FZZZZ5HC2WZZ

### Scenario

RE 2 (Leon B)

### Player Character

Hunk

### What happened?

So this isnt any type of bug or crash, just something that happened near the end that I think probably shouldn't happen. So the part where Mr X would normally throw Ada into the machine and then Leon kisses her....well instead of Ada it was Eveline being kissed. Just kinda creepy, maybe have it set so that the kids in the game cant be in that part?

**Comment by KuronekoLen** [comment]
Just disable them as replacements.
